src/js/components/dashboard/DashboardContent.jsx

```diff
@@ -10,178 +10,199 @@ import DashboardFilters from './DashboardFilters';
 import FiltersMessage from './filters/FiltersMessage';
 
 const propTypes = {
-    loadTableData: PropTypes.func,
-    session: PropTypes.object,
-    activeSubmissions: PropTypes.array,
-    certifiedSubmissions: PropTypes.array,
-    type: PropTypes.string,
-    activeTotal: PropTypes.number,
-    certifiedTotal: PropTypes.number,
-    activeLoading: PropTypes.bool,
-    certifiedLoading: PropTypes.bool,
-    toggleDashboardFilter: PropTypes.func,
-    stagedFilters: PropTypes.object,
-    appliedFilters: PropTypes.object
+  loadTableData: PropTypes.func,
```

Comment:
Some of these changes look like they are just differences in you IDE configuration (e.g. number of spaces indented). See https://github.com/fedspendingtransparency/data-act-documentation/blob/master/frontend_apps/STYLE_GUIDE.md#ide-configuration for style guide and linting plugins

Reply:
IDE Options Updated!